src/modules/release/index.ts

```diff
@@ -33,12 +33,15 @@ export default async (
 ) => {
   checkGit()
   checkIfInGitRepo()
+
+  const normalizedReleaseType = releaseType === 'pre' ? 'prerelease' : releaseType
```

Comment:
A little nitpicking, but wouldn't something along these lines be more general and readable?
```
releaseTypeAliases = {'pre': 'prerelease'} // around line 19
...
releaseType = path(releaseTypeAliases, releaseType) || releaseType
```